usr/share/rear/lib/mkboot-workflow.sh

```diff
@@ -0,0 +1,21 @@
+# mkboot-workflow.sh
+#
+# mkboot workflow for Relax-and-Recover
+#
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+WORKFLOW_mkboot_DESCRIPTION="create boot media without recovery information"
```

Comment:
As far as I know it is "medium" (singular)
versus "media" (plural).

Reply:
1. https://en.wikipedia.org/wiki/Boot_disk talks about media
2. the other help texts also talk about media, not medium

let's keep it like this